devito/symbolics/extended_sympy.py

```diff
@@ -7,14 +7,15 @@
 from sympy import Expr, Integer, Function, Number, Tuple, sympify
 from sympy.core.decorators import call_highest_priority
 
+from devito.finite_differences.elementary import Min, Max
 from devito.tools import Pickable, as_tuple, is_integer
 from devito.types import Symbol
 
 __all__ = ['CondEq', 'CondNe', 'IntDiv', 'CallFromPointer', 'FieldFromPointer',
            'FieldFromComposite', 'ListInitializer', 'Byref', 'IndexedPointer', 'Cast',
            'DefFunction', 'InlineIf', 'Keyword', 'String', 'Macro', 'MacroArgument',
-           'CustomType', 'Deref', 'INT', 'FLOAT', 'DOUBLE', 'VOID', 'CEIL',
-           'FLOOR', 'MAX', 'MIN', 'Null', 'SizeOf', 'rfunc', 'cast_mapper',
+           'CustomType', 'Deref', 'INT', 'FLOAT', 'DOUBLE', 'VOID',
```

Comment:
nit: These 3 ilnes of imports can be 2 I guess

Reply:
Done